sharktank/sharktank/layers/paged_llama_attention_block.py

```diff
@@ -257,10 +257,8 @@ def forward(
         # Used by fp8_e4m3fnuz model
         if self.cache_quantizer is not None:
             if not self.fake_quant:
-                # TODO: this seems like a bastardization of our quantized tensor api
-                # Probably want to add support for using quantized tensors more directly
-                xk = self.cache_quantizer.quantize(xk).unpack().qs
-                xv = self.cache_quantizer.quantize(xv).unpack().qs
+                xk = self.cache_quantizer.quantize_to_raw_tensor(xk)
+                xv = self.cache_quantizer.quantize_to_raw_tensor(xv)
```

Comment:
I think at this point we can just quantize to a quantized tensor instead of to a raw tensor, and rely on unpacking it if necessary wherever its used. This was previously needed because of some weirdness but is probably not worth building on top and providing sharding support to.